scylla-cdc-base/src/test/java/com/scylladb/cdc/model/worker/MockRawChange.java

```diff
@@ -0,0 +1,99 @@
+package com.scylladb.cdc.model.worker;
+
+import com.scylladb.cdc.model.worker.cql.Cell;
+import com.scylladb.cdc.model.worker.cql.Field;
+
+import java.nio.ByteBuffer;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+
+public class MockRawChange implements RawChange, Comparable<MockRawChange> {
+    private final ChangeSchema changeSchema;
+    private final Map<String, Object> columnValues;
+
+    public MockRawChange(ChangeSchema changeSchema, Map<String, Object> columnValues) {
+        this.changeSchema = changeSchema;
+        this.columnValues = columnValues;
+    }
+
+    @Override
+    public ChangeSchema getSchema() {
+        return changeSchema;
+    }
+
+    @Override
+    public Object getAsObject(ChangeSchema.ColumnDefinition c) {
+        return columnValues.get(c.getColumnName());
+    }
+
+    @Override
+    public Cell getCell(ChangeSchema.ColumnDefinition c) {
+        return new Cell() {
+            @Override
+            public ChangeSchema.ColumnDefinition getColumnDefinition() {
+                return c;
+            }
+
+            @Override
+            public Set<Field> getDeletedElements() {
+                throw new UnsupportedOperationException("Not implemented yet");
+            }
+
+            @Override
+            public boolean hasDeletedElements() {
+                throw new UnsupportedOperationException("Not implemented yet");
+            }
+
+            @Override
+            public boolean isDeleted() {
+                throw new UnsupportedOperationException("Not implemented yet");
+            }
+
+            @Override
+            public ByteBuffer getUnsafeBytes() {
+                throw new UnsupportedOperationException("Not implemented yet");
+            }
+
+            @Override
+            public Object getAsObject() {
+                return MockRawChange.this.getAsObject(c);
+            }
+
+            @Override
+            public ChangeSchema.DataType getDataType() {
+                return c.getCdcLogDataType();
+            }
+        };
+    }
+
+    @Override
+    public boolean isNull(ChangeSchema.ColumnDefinition c) {
+        return getAsObject(c) == null;
+    }
+
+    @Override
+    public ByteBuffer getUnsafeBytes(ChangeSchema.ColumnDefinition c) {
+        throw new UnsupportedOperationException("Not implemented yet");
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
Did you consider the `instaceof` trick? We might want to compare `MockRawChange` with `RawChange`

Reply:
`RawChange` is an interface, so you would compare `MockRawChange` with some other implementation of the interface. Currently the only other implementation of `RawChange` is `Driver3RawChange`. But `Driver3RawChange` is in a different module, so `MockRawChange` and `Driver3RawChange` will never "come into contact".